data/scripts/pkmn_center_nurse.inc

```diff
@@ -116,6 +117,10 @@ EventScript_PkmnCenterNurse_GoldCardHealPkmn::
 	goto EventScript_PkmnCenterNurse_HealPkmn
 	end
 
+Movement_PkmnCenterNurse_Turn:
```

Comment:
Why is this no longer just the common movement?

Reply:
Common movements that don't move the player are specially recognized within `applymovement`; there's a comment in `movement.inc` explaining it.

Basically, making the movement be located here has the effect where the nurse turning puts away the follower pokemon

Reply:
And wouldn't it be more clear for users if the follower was put away through a function/macro/command rather than silently depending on a movement? I'd at least explain it with a comment